### ISSUE TITLE
Improving imported callback tests

### ISSUE THREAD

**Issue by queengooborg** [foolip/mdn-bcd-collector#1668]
Digging into the file system API has made me rethink imported callback tests somewhat, and I think I've got an idea for how to simplify them.  While it won't help with the performance as much, it'll help with code construction.

I'm thinking that it would make sense to automatically perform the function wrapping on the build end.  Basically, if a callback function is imported, take the remaining code and wrap it in a function we'll call `after_var()` (`var` being the variable name).  From the perfective of our custom tests file, variables returned through a callback will be no different than a regular variable.

In other words, this:
```js
<%api.GeolocationPosition:posCallback%>
function posCallback(pos) {
  callback(pos.coords);
}
```

...becomes this:
```js
<%api.GeolocationPosition:pos%>
callback(pos.coords);
```

...and the generated code remains roughly the same.

_Originally posted by @queengooborg in https://github.com/foolip/mdn-bcd-collector/pull/1653#discussion_r761015758_
